### PR TITLE
cleanup after manylinux wheelbuilding

### PR DESCRIPTION
**Pull request by fgregg**

### CODE REVIEW

**Comment by mgedmin** [comment]
Strange: the [previous master build](https://travis-ci.org/zopefoundation/zope.index/jobs/224504926) succeeded (and was running `zope-testrunner --test-path=src`), while [this build](https://travis-ci.org/zopefoundation/zope.index/jobs/224826831) fails while running `python setup.py test -q`.  The PR is not modifying `.travis.yml`, so how come the test command is different?

Perhaps you accidentally based your PR on an older commit?  Could you rebase your branch on latest master and force-push?

**Comment by jamadden** [comment]
Yup, that would explain why the PR build worked but the branch build failed. At this moment, the 'manylinux27' branch is 6 commits behind master.

File: .manylinux-install.sh [hunk 1]
```diff
@@ -11,11 +11,11 @@ for PYBIN in /opt/python/*/bin; do
        [[ "${PYBIN}" == *"cp36"* ]]; then
         "${PYBIN}/pip" install -e /io/
         "${PYBIN}/pip" wheel /io/ -w wheelhouse/
+	      rm -rf /io/build /io/*.egg-info
```
Comment (mgedmin) [inline .manylinux-install.sh]: The indentation feels wrong...

File: .manylinux-install.sh [hunk 2]
```diff
@@ -11,11 +11,11 @@ for PYBIN in /opt/python/*/bin; do
...
     fi
 done
 
 # Bundle external shared libraries into the wheels
 for whl in wheelhouse/zope.index*.whl; do
     auditwheel repair "$whl" -w /io/wheelhouse/
-done
-
+done
```
Comment (mgedmin) [inline .manylinux-install.sh]: Please do not remove the trailing newline.

**Comment by mgedmin** [comment]
Ugh, I hate merges that have merge commits.  Any particular reason you didn't git rebase like I asked?

**Comment by fgregg** [comment]
sorry, let me try this again

**Comment by fgregg** [comment]
I'm not sure what's going on with the windows 3.5 builds, but those failures started here  e348415f3ea22b3ea396ab48c22708b1cdafc3d5

**Comment by mgedmin** [comment]
> I don't think the Appveyor failure should block this merge.

Wait, I thought for some reason that this PR didn't touch `appveyor.yml`.  Let me take a look.

**Comment by mgedmin** [comment]
Oh, I see you've fixed Appveyor in #17.

**Comment by mgedmin** [comment]
It looks like you'll need another rebase, since this PR will conflict with #17.  On the plus side, you won't need to touch `appveyor.yml` at all in this PR once #17 is merged.

**Comment by fgregg** [comment]
Could we have a new zope.index release since it seems like there has been a number of changes beyond the manylinux bits.

**Comment by jamadden** [comment]
Will do!

**Pull request merged**